README.md

```diff
@@ -1,8 +1,8 @@
-# `kcp` is a minimal Kubernetes API server
+# `kcp` provides a Kubernetes-like control plane with true multitenancy, with flexible compute powered by real Kubernetes clusters.
```

Comment:
Ok, probably too long:

> kcp provides a true multi-tenant Kubernetes control plane for workloads that span clusters

Reply:
I'm inclined to leave this alone for the time being until we truly can create a workload and have it span multiple clusters, which we can't do (automatically) today. WDYT?

Reply:
We don't need it to span inherently to claim spanning.  We're saying "workloads that span clusters" which is technically true.

And the concern "we don't have it yet" can be an asterisk / clarification down below - since we have to describe current state, future state, and connection anyway.

Reply:
Ok, updated the headline in #576